### PR TITLE
Move multi app install test to Playwright

### PR DESCRIPTION
**Pull request by sgalsaleh**
[SC-112705](https://app.shortcut.com/replicated/story/112705)

### CODE REVIEW

File: e2e/inventory/inventory.go [hunk 1]
```diff
@@ -114,10 +114,11 @@ func NewMultiAppBackupAndRestoreTest() Test {
...
 func MultiAppTest() Test {
 	return Test{
-		Name:        "multi-app-install",
-		TestimSuite: "multi-app-install",
+		ID:          "multi-app-install",
+		Name:        "Multi App Install",
 		Namespace:   "multi-app-install",
-		UpstreamURI: "multi-app-install/automated",
+		AppSlug:     "mutli-app-install",
+		UpstreamURI: "mutli-app-install/automated",
```
Comment (pandemicsyn) [inline e2e/inventory/inventory.go:121]:
```suggestion
		AppSlug:     "multi-app-install",
		UpstreamURI: "multi-app-install/automated",
```

Comment (pandemicsyn) [reply e2e/inventory/inventory.go:121]: @sgalsaleh this looks like a typo - but i see the license and playwright tests are looking for `mutli` as well so not positive.

**Review by pandemicsyn — COMMENTED** [review]
~~@sgalsaleh I think maybe the test got skipped during the action run, but not immediately obvious to me why.~~

just saw the update.

File: e2e/inventory/inventory.go (hunk 1 above)
Comment (sgalsaleh) [reply e2e/inventory/inventory.go:121]: yeah, it's a typo in the app slug, but this matches so it should work...

**Review by pandemicsyn — APPROVED** [review]

**Pull request merged**